src/payment/strategies/stripev3/stripev3.ts

```diff
@@ -41,6 +41,21 @@ export interface PaymentIntent {
      * Status of this PaymentIntent. Read more about each PaymentIntent [status](https://stripe.com/docs/payments/intents#intent-statuses).
      */
     status: 'succeeded' | string;
+
+     /**
+      * The payment error encountered in the previous PaymentIntent confirmation.
+      */
+     last_payment_error?: LastPaymentError;
```

Comment:
Are these fields all optional?

https://stripe.com/docs/api/errors#errors-payment_intent-last_payment_error

Reply:
only `last_payment_error` could not be present in the PaymentIntent.

Reply:
Hey @BC-SRodriguez, wanted to ask one thing. Is the above field optional/undefined or it it null. Since stripe's own 
typings set it to be `null` https://github.com/stripe/stripe-node/blob/e57cf41c5a309e11cce9918e308ad7935f2f5566/types/2020-08-27/PaymentIntents.d.ts#L107

Reply:
Hi @animesh1987, `last_payment_error` could be null. I have made the changes according to https://github.com/stripe/stripe-node/blob/e57cf41c5a309e11cce9918e308ad7935f2f5566/types/2020-08-27/PaymentIntents.d.ts#L107